db/migrate/20190513155251_add_price_to_service_templates.rb

```diff
@@ -0,0 +1,6 @@
+class AddPriceToServiceTemplates < ActiveRecord::Migration[5.0]
+  def change
+    add_reference :service_templates, :currency, :type => :bigint
```

Comment:
I think we should have `chargeback_rate_detail_currency` here - to see that it is related to `chargeback_rate_detail_currencies` table.

`add_reference :service_templates, :chargeback_rate_detail_currency, :type => :bigint`

you can specify foreign key in the model.

@carbonin what do you think ?

Reply:
I think the idea is to make the method `#currency` and the column `currency_id` so the relationship in the model specifies the class https://github.com/ManageIQ/manageiq/pull/18754/files#diff-9257fff45a7d1258305c9a6b7d15e322R66

Reply:
It's unfortunate that `chargeback_rate_detail_currency` isn't named just `currencies` because it doesn't look like the table has anything chargeback specific in it.

Reply:
@carbonin Exactly what I am going to say.

Reply:
👍 👍

Reply:
What is the effort to rename the table :)